api_tests/dry/multiple_peers.ts

```diff
@@ -82,6 +82,21 @@ import "../lib/setup_chai";
             aliceSwapWithBobHref.should.be.a("string");
```

Comment:
Isn't there more e2e test simplifications? For the test where we look for the swap on Bob side after Alice sent it?

Reply:
Probably, I'll create a follow-up ticket for that if you don't mind!

Reply:
Actually, this is going away with the new e2e tests so I think we don't need to change it now :)